### PR TITLE
bot.getArticleInfo()

### PR DESCRIPTION
**Pull request by Technical-13**
Add this to the baked in command list. :)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+	"author": "Technical-13 <Technical_13@yahoo.com> (https://magentarv.info/technical-13)",
 	"description": "MediaWiki API client written in node.js",
 	"main": "./lib/bot.js",
 	"repository": {
 		"type": "git",
-		"url": "git://github.com/macbre/nodemw.git"
+		"url": "git://github.com/Technical-13/nodemw.git"
 	},
 	"keywords": [
 		"mediawiki",
```
Comment (macbre) [inline package.json]: @Technical-13 - can you revert these changes? Feel free to add yourself to the `authors` 🙂

**Comment by macbre** [comment]
@Technical-13 - please fix linting errors - you can run `npm run lint` locally.

**Review by macbre — CHANGES_REQUESTED** [review]
Thanks for these changes. Please fix issues reported by `eslint`.

File: package.json (hunk 1 above)
Comment (Technical-13) [reply package.json]: Done.

**Comment by macbre** [comment]
@Technical-13 - please run `npx eslint . --fix` - there are still some linting issues.

**Comment by Technical-13** [comment]
Done.

File: README.md [hunk 2]
```diff
@@ -182,6 +182,10 @@ Gets all revisions of a given article - [read more](http://www.mediawiki.org/wik
 
 Gets all categories a given article is in - [read more](http://www.mediawiki.org/wiki/API:Property/Categories)
 
+### bot.getArticleInfo(title, callback)
+
+Gets all info of a given article - [read more](https://www.mediawiki.org/wiki/API:Info)
+
```
Comment (macbre) [inline README.md:188]: 👍🏻

**Review by macbre — APPROVED** [review]

**Pull request merged**